crates/fuel-core/src/p2p_test_helpers.rs

```diff
@@ -83,6 +83,25 @@ pub enum BootstrapType {
     ReservedNodes,
 }
 
+/// Set of values that will be overridden in the node's configuration
+#[derive(Clone, Debug)]
+pub struct ConfigOverrides {
```

Comment:
[nit] I like more the name `CustomizeConfig`

Reply:
addressed in e676fcc233